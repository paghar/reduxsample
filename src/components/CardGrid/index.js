import React, { useState } from "react"
import { Main, Cards, CardsItem } from "./index.style"
import Card from "../Card"
import Button from "../Button"

const CardGrid = ({ CardsData, updateList, randomRatingButton }) => {
  const [isStartTimer, setStartTimer] = useState(true)
  const [timer, setTimer] = useState(null)

  function randomRating() {
    setStartTimer(!isStartTimer)
    if (isStartTimer) {
      setTimer(
        setInterval(() => {
          randomRatingButton()
        }, 3000)
      )
    } else {
      clearInterval(timer)
    }
  }

  return (
    <Main>
      <Button
        clicked={() => randomRating()}
        isStartTimer={isStartTimer}
      ></Button>
      <Cards>
        {CardsData.map(item => (
          <CardsItem key={item.id}>
            <Card
              image={item.image}
              title={item.title}
              context={item.context}
              rateValue={item.rate}
              clicked={value => updateList(value, item.id)}
            />
          </CardsItem>
        ))}
      </Cards>
    </Main>
  )
}

export default CardGrid
