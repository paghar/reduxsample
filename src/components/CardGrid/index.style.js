import styled from "styled-components"

export const Main = styled.div`
  margin-top: 150px;
`

export const Cards = styled.ul`
  display: flex;
  flex-wrap: wrap;
  list-style: none;
  margin: 0;
  padding: 0;
`

export const CardsItem = styled.li`
  width: 25%;
  padding: 1rem;
`
