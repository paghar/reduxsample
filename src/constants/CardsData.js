import Image1 from "../assets/image/1.jpg"
import Image2 from "../assets/image/2.jpg"
import Image3 from "../assets/image/3.jpg"
import Image4 from "../assets/image/4.jpg"
import Image5 from "../assets/image/5.jpg"
import Image6 from "../assets/image/6.jpg"
import Image7 from "../assets/image/7.jpg"
import Image8 from "../assets/image/8.jpg"
import Image9 from "../assets/image/9.jpg"
import Image10 from "../assets/image/10.jpg"
import Image11 from "../assets/image/11.jpg"
import Image12 from "../assets/image/12.jpg"
import Image13 from "../assets/image/13.jpg"
import Image14 from "../assets/image/14.jpg"
import Image15 from "../assets/image/15.jpg"
import Image16 from "../assets/image/16.jpg"
import Image17 from "../assets/image/17.jpg"
import Image18 from "../assets/image/18.jpg"
import Image19 from "../assets/image/19.jpg"
import Image20 from "../assets/image/20.jpg"
import Image21 from "../assets/image/21.jpg"
import Image22 from "../assets/image/22.jpg"
import Image23 from "../assets/image/23.jpg"
import Image24 from "../assets/image/24.jpg"
export const CardsData = [
  {
    id: 1,
    image: Image1,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 5
  },
  {
    id: 2,
    image: Image2,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 1
  },
  {
    id: 3,
    image: Image3,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 2
  },
  {
    id: 4,
    image: Image4,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 3
  },
  {
    id: 5,
    image: Image5,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 4
  },
  {
    id: 6,
    image: Image6,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 5
  },
  {
    id: 7,
    image: Image7,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 2
  },
  {
    id: 8,
    image: Image8,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 2
  },
  {
    id: 9,
    image: Image9,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 1
  },
  {
    id: 10,
    image: Image10,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 1
  },
  {
    id: 11,
    image: Image11,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 4
  },
  {
    id: 12,
    image: Image12,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 0
  },
  {
    id: 13,
    image: Image13,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 4
  },
  {
    id: 14,
    image: Image14,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 3
  },
  {
    id: 15,
    image: Image15,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 2
  },
  {
    id: 16,
    image: Image16,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 1
  },
  {
    id: 17,
    image: Image17,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 4
  },
  {
    id: 18,
    image: Image18,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 3
  },
  {
    id: 19,
    image: Image19,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 2
  },
  {
    id: 20,
    image: Image20,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 2
  },
  {
    id: 21,
    image: Image21,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 3
  },
  {
    id: 22,
    image: Image22,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 5
  },
  {
    id: 23,
    image: Image23,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 2
  },
  {
    id: 24,
    image: Image24,
    title: "Chicken fried steak",
    context:
      "chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas",
    rate: 1
  }
]
