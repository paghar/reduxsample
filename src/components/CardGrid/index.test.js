import React from "react"
import renderer from "react-test-renderer"
import CardGrid from "."
import { CardsData } from "../../constants/CardsData"

describe("<CardGrid />", () => {
  it("CardGrid", () => {
    const tree = renderer.create(<CardGrid CardsData={CardsData} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
