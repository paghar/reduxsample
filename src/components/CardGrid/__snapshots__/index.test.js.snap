// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardGrid /> CardGrid 1`] = `
<div
  className="sc-bdVaJa fmgygi"
>
  <button
    className="sc-dnqmqq jizNsI"
    onClick={[Function]}
  >
    <span>
      Start Random Rating
       
    </span>
  </button>
  <ul
    className="sc-bwzfXH ifRnlg"
  >
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="1.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="2.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="3.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="4.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="5.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="6.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="7.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="8.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="9.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="10.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="11.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="12.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="13.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="14.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="15.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="16.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="17.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="18.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="19.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="20.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="21.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="22.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="23.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
    <li
      className="sc-htpNat jwqEMN"
    >
      <div
        className="sc-bxivhb ezUWkz"
      >
        <div
          className="sc-ifAKCX ejNwWU"
        >
          <img
            className="sc-bZQynM epMubi"
            src="24.jpg"
          />
          <h2
            className="sc-EHOje bVAOkt"
          >
            Chicken fried steak
          </h2>
          <p
            className="sc-gzVnrw kmmynC"
          >
            chicken fried steak was born to go with American food classics like mashed potatoes and black-eyed peas
          </p>
          <div
            className="sc-htoDjs fKhaWj"
          >
            <div
              className="star-rating"
            >
              <div
                className="star selected"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
              <div
                className="star"
                onClick={[Function]}
              />
            </div>
          </div>
        </div>
      </div>
    </li>
  </ul>
</div>
`;
